extension/landing-zone/variables.tf

```diff
@@ -0,0 +1,324 @@
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to provision resources to"
+  type        = string
+  sensitive   = true
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
+  default     = null
+}
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "au-syd"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "slz-vsi"
+}
+
+variable "vpc_id" {
+  description = "Id of the VPC in which the VSI will be created."
+  type        = string
+  default     = "r026-9c433ed0-69f5-4554-ab35-c86d01ef1b6c"
+}
+
+variable "ssh_key" {
+  type        = string
+  description = "An existing ssh key name to use for this example, if unset a new ssh key will be created"
+  default     = null
+}
+
+variable "resource_tags" {
+  description = "List of Tags for the resource created"
+  type        = list(string)
+  default     = null
+}
+
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the VSI resources created by the module."
+  default     = []
+}
+
+variable "image_name" {
+  description = "Image ID used for VSI. Run 'ibmcloud is images' to find available images. Be aware that region is important for the image since the id's are different in each region."
+  type        = string
+  default     = "ibm-ubuntu-18-04-6-minimal-amd64-2"
```

Comment:
when this is merged, please reach out to @MatthewLemmond to update our current automation that updates this image ID. 

FYI, latest image is `ibm-ubuntu-22-04-2-minimal-amd64-1` so we should use this as the default for now